src/main.cpp

```diff
@@ -2241,6 +2241,15 @@ void vcRenderWindow(vcState *pProgramState)
         case vcWL_SceneLeft:
           ImGui::Columns(2);
 
+          if (!pProgramState->settings.presentation.columnSizeCorrect)
+          {
+            ImGui::SetColumnWidth(0, size.x - pProgramState->settings.presentation.sceneExplorerSize);
+            pProgramState->settings.presentation.columnSizeCorrect = true;
+          }
+
+          if (ImGui::GetColumnWidth(0) != size.x - pProgramState->settings.presentation.sceneExplorerSize)
```

Comment:
`float` comparison. You sure this will work?

Reply:
Worst case it does the if every time which does nothing anyway